app/screens/home/profile_screen.tsx

```diff
@@ -0,0 +1,201 @@
+import React from "react";
+import { View, Text } from "react-native";
+import { ThemedIconButton } from "@/components/theme/ThemedIconButton";
+import { ThemedText } from "@/components/theme/ThemedText";
+import { ThemedView } from "@/components/theme/ThemedView";
+import { TopBar } from "@/components/navigation/TopBar";
+import { TouchableOpacity } from "react-native";
+import { Image } from "react-native";
+import * as ImagePicker from 'expo-image-picker';
+import { Dimensions } from "react-native";
+import { ThemedTextButton } from "@/components/theme/ThemedTextButton";
+import { BottomBar } from "@/components/navigation/BottomBar";
+import { Alert } from "react-native";
+import { Icon } from "react-native-elements";
+import { auth } from "@/firebase/Firebase"; 
+import { updateEmail, sendPasswordResetEmail} from "firebase/auth";
+
+
+//TODO : change the colors for light mode 
```

Comment:
What's this comment corresponding to ?

Reply:
Just because I didn't finished the UI for light screen